Src/BootCamp.Chapter1/ArrayOperations.cs

```diff
@@ -40,7 +76,20 @@ public static int[] RemoveLast(int[] array)
         public static int[] RemoveFirst(int[] array)
         {
             // ToDo: implement.
-            return array;
+            if (array != null || array.Length >= 1)
```

Comment:
![Codacy](https://app.codacy.com/assets/images/favicon.png) Issue found: [Either reverse the equality operator in the 'array' null test, or reverse the logical operator that follows it.](https://app.codacy.com/manual/Almantask/CSharp-From-Zero-To-Hero/pullRequest?prid=4940124)